admin/javascript/LeftAndMain.Panel.js

```diff
@@ -134,5 +134,14 @@
 				return false;
 			}
 		});
+
+		$('.cms-content-tools').entwine({
```

Comment:
Haha good spot. I think I had originally placed the code in a different file and then copy/pasted it. I'll refactor and re-push.